iamctl/pkg/utils/keywordUtils.go

```diff
@@ -149,6 +149,10 @@ func resolvePathWithIdentifiers(arrayName string, element interface{}, identifie
 		}
 	}
 	identifier := identifiers[arrayName]
+	if resourceType == "Claims" && arrayName == "properties" {
```

Comment:
Instead of hardcoding this inside this method, let's have separate arrayIdentifiers for each resource type and pass the relevant identifier into the resolvePathWithIdentifiers method.

Reply:
Added separate arrayIdentifiers for each resource type.